blueman/plugins/manager/Notes.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import os
+from gettext import bind_textdomain_codeset
+from tempfile import NamedTemporaryFile
+
+from blueman.Functions import create_menuitem, get_icon, launch, UI_PATH
+from blueman.plugins.ManagerPlugin import ManagerPlugin
+
+import gi
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
+
+
+def send_note_cb(dialog, response_id, device_address, text_view):
+    text = text_view.get_buffer().props.text
+    dialog.destroy()
+    if response_id == Gtk.ResponseType.CANCEL:
+        return
+
+    date = datetime.datetime.now().strftime('%Y%m%dT%H%M00')
+    data = ('BEGIN:VNOTE \n'
+            'VERSION:1.1 \n'
+            'BODY;CHARSET=UTF-8: %s \n'
+            'DCREATED:%s \n'
+            'LAST-MODIFIED:%s \n'
+            'CLASS:PUBLIC \n'
+            'X-IRMC-LUID:000001000000 \n'
+            'END:VNOTE \n' % (' '.join(text.splitlines()), date, date))
+
+    tempfile = NamedTemporaryFile(delete=False)
```

Comment:
Not an issue but the ``tempfile`` module supports the ``with`` statement, which is convenient as it cleans up after itself :smile: .

```python
with NamedTemporaryile(delete=False) as tempfile:
    do_stuff
```